src/Containers.scss

```diff
@@ -33,3 +33,19 @@
 .ct-table-empty td {
     padding: var(--pf-global--spacer--sm) var(--pf-global--spacer--md) var(--pf-global--spacer--lg);
 }
+
+// Fix label color and alignment in the table cell
+.container-state {
+  // Container states: configured created running stopped paused exited removing
+  &-configured, &-created, &-stopped, &-exited {
```

Comment:
Since we add color to only 'running' can we just turn this around and remove it if it is not `running`? Seems easier to maintain and less likely to break with some state we don't count on.

Reply:
We don't add any color. It uses the "green" PF property, specified in the JS, and PF handles the style. We want these specific states to be transparent.

We could add a transparent state in the JS, but I didn't want to specify style in the code; I wanted to keep it separate.